src/style.css

```
@@ -1 +1,68 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--1 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--2 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--3 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--4 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--5 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
  background-image: url(./images/star-active.svg);
}
```

Combine selectors with identical styles using comma